### ISSUE TITLE
Reproducibility: follow up

### ISSUE THREAD

**Issue by alokendumazumder** [paganpasta/coss-pytorch#3]
Hi @paganpasta . Thank you for getting back to me so quickly.

As I am new to the area of knowledge distillation, I have a few queries:

The link of the weights, I believe they are students's weights, right? I have loaded the teacher's weights from the SEED repo.

To reproduce Table 1, I believe you have trained (distilled from the teacher) student from Imagenet1K (In Imagenet1K there are three training files, task1, task2, and task3, here task1 and task2 are merged as a single trainfile, I guess you have used the task1 and task2 ?). Are the results classified on the val set ?

For Table2, you have used the pre-trained weights of students (distilled from students in Tab 1), and performed classification on Cifar and other datasets, right?

Thank you

**Comment by paganpasta** [comment]
Hi @alokendumazumder 
Yes, the weights are for the student. 

> To reproduce Table 1, I believe you have trained (distilled from the teacher) student from Imagenet1K (In Imagenet1K there are three training files, task1, task2, and task3, here task1 and task2 are merged as a single trainfile, I guess you have used the task1 and task2 ?). Are the results classified on the val set ?

I am unsure about the structure youd have on your side, as it depends on the place you get the data from. I'd recommend to follow the standard structure for ImageNet which I think you can get from older papers about Image Classification. 

> For Table2, you have used the pre-trained weights of students (distilled from students in Tab 1), and performed classification on Cifar and other datasets, right?

Yes, we do not modify the student weights but only learn a classification head on top.
